projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -279,6 +279,21 @@ protected function __construct() {
 			)
 		);
 
+		// Add "temp" as a post status for temporary storage when saveResponses is 'no'
+		register_post_status(
+			'temp',
```

Comment:
This worked fine, I just wonder if we should purposely use a less common word. After all, this will live both on dotcom and on any self hosted, we need to make sure the post status registration doesn't fail because there was already one there with the same name. I'd use some more complex, like jetpack-temp-feedback or something